src/store/ChartStore.js

```diff
@@ -274,6 +283,53 @@ class ChartStore {
         this.feed.onComparisonDataUpdate(this.updateComparisons);
     }
 
+
+    /**
+     * Get tradeTimes if not loaded yet
+     * OR update the active symbols by comapring open time
+     */
+    @action.bound updateMarketClosedStatus(api) {
+        const todayUtc = (new Date()).getTime();
+        const timeToEpochGMT = (hour) => {
+            const dateStr = new Date().toISOString().substring(0, 11);
+            return new Date(`${dateStr}${hour}Z`).getTime();
+        };
+
+
+        if (this.tradeTimes.length && this.tradeTimesCapture.getDate() === (new Date()).getDate()) {
+            this.tradeTimes.forEach((market) => {
+                market.submarkets.forEach((submarket) => {
+                    submarket.symbols.forEach((symbol) => {
+                        const foundSymbol = this.activeSymbols.find(item => item.symbol === symbol.symbol);
+                        if (foundSymbol) {
+                            let isOpen = false;
+                            for (let i = 0; i <= symbol.times.open.length; i++) {
```

Comment:
considering how often you access it, you could take it out to make it look cleaner:

```js
const { open, close } = symbol.times;
```

Reply:
@shayanbinary this line could be simplified to just `for (let i = 0; i <= open.length; i++) {`